src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -339,6 +339,7 @@ public virtual Type[] GetGenericParameterConstraints()
         [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties | DynamicallyAccessedMemberTypes.NonPublicProperties)]
         public abstract PropertyInfo[] GetProperties(BindingFlags bindingAttr);
 
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
Why All (that includes NestedTypes as well) ? The doc says it can be only `constructor, method, property, or field` and I think they have to be public